calculator.py

```diff
@@ -0,0 +1,51 @@
+print("Calculator")
+result = 0
+run_again = True
+
+def calculate():
+    # global keyword allows us to modify the variable outside of the current scope
+    global result
```

Comment:
**praise**: Good to capture the purpose. Shows why `result` is defined as global